add feature blog 1y support (#23582)
kubernetes/website: content/en/blog/_posts/2020-08-31-increase-kubernetes-support-one-year.md

A survey conducted in early 2019 by the WG LTS showed that a significant subset of Kubernetes end-users fail to upgrade within the 9-month support period. 

![Versions in Production](/images/blog/2020-08-31-increase-kubernetes-support-one-year/versions-in-production-text-2.png)

This, and other responses from the survey, suggest that a considerable portion of our community would better be able to lifecycle manage their deployments on supported versions if the patch support period were extended to 12-14 months. It appears to be true regardless of whether the users are on DIY build or commercially vendored distributions. An extension in the patch support length of time would thus lead to a larger percentage of our user base running supported versions compared to what we have now.

> A yearly support period provides the cushion end-users appear to desire, and is more in harmony with familiar annual planning cycles.

Reviewer: ```suggestion
A yearly support period provides the cushion end-users appear to desire, and is more aligned with familiar annual planning cycles.
```

Feel free to disregard, just thought aligned sounds better in this instance.

> We view elongating the patch support window toward 1 year as a relatively minor change, yet is believed viable from an implementation perspective and is a meaningful first response to users.

Reviewer: ```suggestion
```
I would remove this sentence, it doesn't flow well and seems a little redundant with the previous sentence.

> There are many unknowns about changing the support windows for a project with as many moving parts as Kubernetes, and keeping the change relatively small (relatively being the important word), gives us the chance to find out what those unknowns are in detail and address them.

Reviewer: ```suggestion
There are many unknowns about changing the support windows for a project with as many moving parts as Kubernetes. Keeping the change relatively small (relatively being the important word), gives us the chance to find out what those unknowns are in detail and address them.
```
Broke it into 2 sentences

> From Kubernetes version 1.19 on, the support window will be extended to one year.For Kubernetes versions 1.16, 1.17, and 1.18, the story is more complicated.

Reviewer: ```suggestion
From Kubernetes version 1.19 on, the support window will be extended to one year. For Kubernetes versions 1.16, 1.17, and 1.18, the story is more complicated.
```
missing space after `one year.`

> All of these versions still fall under the older “three releases support” model, and so will be out of support when 1.19, 1.20 and 1.21 are respectively released. However, because the 1.19 release has been delayed due to the events of 2020, they will end up with close to a year of support (depending on their exact release dates).

Reviewer: ```suggestion
All of these versions still fall under the older “three releases support” model, and will drop out of support when 1.19, 1.20 and 1.21 are respectively released. However, because the 1.19 release has been delayed due to the events of 2020, they will end up with close to a year of support (depending on their exact release dates).
```
Thought this flowed better, feel free to ignore 👍

> For example, 1.19 has been released on the 26th of August 2020, which is 11 months since the release of 1.16. Since 1.16 is still under the old release policy, this means that it is now out of support.

Reviewer: ```suggestion
For example, 1.19 was released on the 26th of August 2020, which is 11 months since the release of 1.16. Since 1.16 is still under the old release policy, this means that it is now out of support.
```
Similar, just felt this flowed a bit better - feel free to disregard.
